components/XOGame.js

```diff
@@ -0,0 +1,140 @@
+import styled from "styled-components";
+import {useState} from "react";
+
+const players = {
+
+  HUMAN: {
+  SYM: "X",
+  NAME: "You",
+  },
+};
+
+export default function XOGame() {
+  // const [board, setBoard] = useState(Array(9).fill(""));
+  const [board, setBoard] = useState([
+    ["", "", ""],
+    ["", "", ""],
+    ["", "", ""],
+  ]);
+
+  function playFn(arrayIndex, index) {
```

Comment:
your functions should have more descriptive names, so that other developers know, what the function actually does